### ISSUE TITLE
Problem uploading image into Media Field

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#367]
```
What steps will reproduce the problem? (Be specific enough so that someone
else can make the plugin fail for themselves. Remember: if the issue can't
be fixed if it can't be reproduced!)
1. added custom field 'Media File 1' of type 'Media Field'. It's not an array 
and I use the default output filter image_src
2. added the field to built-in post type 'page' and activated it
3. when adding a new page, the custom field appears and I'm given two buttons: 
'Select' (that's what I named the button) and 'Upload'
4. The 'Select' button works fine. When choosing 'Upload' instead, I upload a 
new image and then I click 'Insert into Post'

What is the expected output? What do you see instead?

After clicking 'Insert into Post' I am returned to the draft 'page' I was 
editing and an error message 'Post ID or guid required' is displayed below the 
'Select/Upload' buttons, and no image is assigned to the custom field. 

The same happens when I click 'Upload' and pick an image from the Media Library 
instead (i.e. I don't upload a new one). 


Does the problem continue if you disable all other plugins? (Conflicts
between plugins are easily the #1 cause of errors, so it's important to
isolate the issue):

Yes.

Please copy and paste the system information that appears in a yellow
textarea when you click on "Report a Bug" on any CCTM admin screen (this
includes the version of the
plugin, the version of PHP, the version of MySQL, a list of other active
plugins etc.):


*SYSTEM INFO* 
------------------------ 
Plugin Version: 0.9.5.11-pl
WordPress Version: 3.3.1
PHP Version: 5.3.3-7+squeeze8
MySQL Version: 5.1.49-3
Server OS: Linux
------------------------ 
ACTIVE PLUGINS: 
------------------------ 
CURRENT THEME: Twenty Eleven v.1.3 http://wordpress.org/
```

Original issue reported on code.google.com by `m.fabb...@gmail.com` on 4 Apr 2012 at 10:22

Attachments:
- [Clipboard02.jpg](https://storage.googleapis.com/google-code-attachments/wordpress-custom-content-type-manager/issue-367/comment-0/Clipboard02.jpg)

**Comment by GoogleCodeExporter** [comment]
Original comment by `ever...@fireproofsocks.com` on 24 Apr 2012 at 5:45
- Changed state: **InReview**

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed on the dev branch:

Committed revision 536819.

Will be released as 0.9.5.12 shortly.
```

Original comment by `ever...@fireproofsocks.com` on 26 Apr 2012 at 3:53
- Changed state: **Fixed**
